src/test/ui/issues/issue-7813.stderr

```diff
@@ -2,9 +2,12 @@ error[E0282]: type annotations needed for `&[_; 0]`
   --> $DIR/issue-7813.rs:2:13
    |
 LL |     let v = &[];
-   |         -   ^^^ cannot infer type
-   |         |
-   |         consider giving `v` the explicit type `&[_; 0]`, with the type parameters specified
+   |             ^^^ cannot infer type
+   |
+help: consider giving this binding the explicit type `&[_; 0]`, with the type parameters specified
+   |
+LL |     let v: &[_; 0] = &[];
+   |          ^^^^^^^^^
```

Comment:
Arguably we should have been suggesting a slice instead of a borrowed empty array, but would like to address that in a separate PR.

Reply:
Resolved in the last commit.